salesagility/SuiteCRM
# Fix #2645 - Calendar quick create ignores required fields

## Description
Addresses issue #2645

## How To Test This
1. Go to calendar
2. Click in calendar to create new meeting
3. Click Save (without filling in required field "Subject")
4. See that it now required before saving

## Types of changes


### Final checklist
